src/components/modals/PermissionsSettingModal.tsx

```diff
@@ -38,7 +38,7 @@ export function PermissionsSettingModal(props: {
   )
 
   const handlePress = () => {
-    openSettings().catch(showError)
+    openSettings().catch(showDevError)
```

Comment:
Change this back to showError, but do it correctly so we get a stack trace: `.catch(error => showError(error))`